example/assets/mock-data/record.json

```diff
@@ -1,878 +1,854 @@
-        ],
-        "number": 29,
-        "publication_info": {
-          "year": 2014
         }
-      }
-    },
-    {
-      "reference": {
-        "collaboration": [
-          "ATLAS Collaboration"
-        ],
-        "misc": [
-          "Analysis of events with b-jets and a pair of leptons of the same charge in pp collisions at \u221a s = 8 TeV with the ATLAS detector"
-        ],
-        "number": 30,
-        "publication_info": {
-          "year": 2015
+      }, 
+      {
+        "reference": {
+          "collaboration": [
+            "ATLAS Collaboration"
+          ], 
+          "misc": [
+            "Analysis of events with b-jets and a pair of leptons of the same charge in pp collisions at \u221a s = 8 TeV with the ATLAS detector"
+          ], 
+          "number": 30, 
+          "publication_info": {
+            "artid": "150", 
+            "journal_title": "JHEP", 
+            "journal_volume": "1510", 
+            "page_start": "150", 
+            "year": 2015
+          }
         }
-      }
-    },
-    {
-      "reference": {
-        "collaboration": [
-          "ATLAS Collaboration"
-        ],
-        "misc": [
-          "Search for production of vector-like quark pairs and of four top quarks in the lepton-plus-jets final state in pp collisions at \u221a s = 8 TeV with the ATLAS detector"
-        ],
-        "number": 31,
-        "publication_info": {
-          "year": 2015
+      }, 
+      {
+        "reference": {
+          "collaboration": [
+            "ATLAS Collaboration"
+          ], 
+          "misc": [
+            "Search for production of vector-like quark pairs and of four top quarks in the lepton-plus-jets final state in pp collisions at \u221a s = 8 TeV with the ATLAS detector"
+          ], 
+          "number": 31, 
+          "publication_info": {
+            "artid": "105", 
+            "journal_title": "JHEP", 
+            "journal_volume": "1508", 
+            "page_start": "105", 
+            "year": 2015
+          }
         }
       }
-    }
-  ],
-  "report_numbers": [
-    {
-      "value": "ATL-PHYS-PROC-2015-144"
-    }
-  ],
-  "self": {
-    "$ref": "http://localhost:5000/api/literature/1404685"
-  },
-  "self_recid": 1404685,
-  "titles": [
-    {
-      "source": "arXiv",
-      "title": "Top-quark production measurements"
-    }
-  ],
-  "urls": [
-    {
-      "value": "http://inspirehep.net/record/1404685/files/arXiv:1511.04200.pdf"
-    }
-  ]
-}
\ No newline at end of file
+    ], 
+    "report_numbers": [
+      {
+        "value": "ATL-PHYS-PROC-2015-144"
+      }
+    ], 
+    "self": {
+      "$ref": "http://localhost:5000/api/literature/1404685"
+    }, 
+    "titles": [
+      {
+        "source": "arXiv", 
+        "title": "Top-quark production measurements"
+      }
+    ], 
+    "urls": [
+      {
+        "value": "http://inspirehep.net/record/1404685/files/arXiv:1511.04200.pdf"
+      }
+    ]
+  }
```

Comment:
the records are for same `recid` but this one is from `db` and since schema has changed, diff is a little bigger than usual.